### PR TITLE
fixes some grammar for eating things

### PR DESCRIPTION
**Pull request by alsoandanswer**
- rscadd: "Food that takes one bite to eat doesn't say 'you finish it' anymore. Sounded kind of weird to 'finish' something in one bite."
  - bugfix: "Fixes some grammar with swallowing pills."

yeah that's about it

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -32,11 +32,17 @@
 /obj/item/reagent_containers/food/snacks/on_consume(mob/user, mob/target)
 	if(!reagents.total_volume && !trash)
 		target.visible_message("<b>[target]</b> finishes [is_liquid ? "drinking" : "eating"] \a [src].",
-					 SPAN_NOTICE("You finish [is_liquid ? "drinking" : "eating"] \a [src]."))
+				   SPAN_NOTICE("You finish [is_liquid ? "drinking" : "eating"] \a [src]."))
+		if(src.reagents_to_add == bitesize)
+			target.visible_message("<b>[target]</b> finishes [is_liquid ? "drinking" : "eating"] \a [src].",
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/snacks.dm:37]: isn't this going to make two visible messages

Comment (alsoandanswer) [reply code/modules/reagents/reagent_containers/food/snacks.dm:37]: oops

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 2]
```diff
@@ -116,13 +116,13 @@
 	reagents_to_add = list(/datum/reagent/diethylamine = 60)
 
 /obj/item/reagent_containers/glass/bottle/pacid
-	name = "Polytrinic Acid Bottle"
+	name = "polytrinic Acid Bottle"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/glass/bottle.dm:119]:
```suggestion
	name = "polytrinic acid bottle"
```

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 3]
```diff
@@ -116,13 +116,13 @@
...
 	desc = "A small bottle. Contains a small amount of Polytrinic Acid"
 	icon_state = "bottle-4"
 	reagents_to_add = list(/datum/reagent/acid/polyacid = 60)
 
 /obj/item/reagent_containers/glass/bottle/adminordrazine
-	name = "Adminordrazine Bottle"
+	name = "adminordrazine Bottle"
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/glass/bottle.dm:125]:
```suggestion
	name = "adminordrazine bottle"
```

File: code/modules/reagents/reagent_containers/pill.dm [hunk 4]
```diff
@@ -65,7 +65,7 @@
 
 		if(proximity && target.is_open_container() && target.reagents)
 			if(!target.reagents.total_volume)
-				to_chat(user, SPAN_NOTICE("[target] is empty. Can't dissolve \the [src]."))
+				to_chat(user, SPAN_NOTICE("You can't dissolve \the [src] in the empty [target]."))
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/pill.dm:68]:
```suggestion
				to_chat(user, SPAN_NOTICE("You can't dissolve \the [src] in an empty [target]."))
```

**Pull request merged**